pontoon/settings/base.py

```diff
@@ -106,11 +106,15 @@ def path(*args):
 RAYGUN4PY_CONFIG = {"api_key": os.environ.get("RAYGUN_APIKEY", "")}
 
 # Email settings
-EMAIL_HOST_USER = os.environ.get("SENDGRID_USERNAME", "")
-EMAIL_HOST = "smtp.sendgrid.net"
-EMAIL_PORT = 587
-EMAIL_USE_TLS = True
-EMAIL_HOST_PASSWORD = os.environ.get("SENDGRID_PASSWORD", "")
+EMAIL_HOST_USER = os.environ.get(
+    "EMAIL_HOST_USER", os.environ.get("SENDGRID_USERNAME", "")
```

Comment:
According to [Sendgrid docs](https://sendgrid.com/docs/for-developers/sending-email/upgrade-your-authentication-method-to-api-keys/), the default value can now be hardcoded to "apikey".

Please also update the deployment docs accordingly.